ci-operator/jobs/openshift/release/openshift-release-periodics.yaml

```diff
@@ -115,3 +114,60 @@ periodics:
             name: cluster-secrets-gcp
         - configMap:
             name: cluster-profile-gcp
+- agent: kubernetes
+  cron: '@yearly'
```

Comment:
oh boy

Reply:
same as the others - these are templates intended to be invoked from external automation.  I.e. until we have another prow job type.